fe1-web/src/features/evoting/reducer/ElectionKeyReducer.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * This error is disabled since reducers use the createSlice function, which requires the user to
+ * param-reassign. Please do not disable other errors.
+ */
+/* eslint-disable no-param-reassign */
+
+import { createSelector, createSlice, Draft, PayloadAction } from '@reduxjs/toolkit';
+
+import { PublicKey } from 'core/objects';
+
+/**
+ * Reducer & associated functions to store received election keys
+ */
+
+export interface ElectionKeyReducerState {
+  byElectionId: {
+    [electionId: string]: string;
+  };
+}
+
+const initialState: ElectionKeyReducerState = {
+  byElectionId: {},
+};
+
+export const ELECTION_KEY_REDUCER_PATH = 'electionKey';
```

Comment:
why have a separate reducer from the `election` reducer ?

Reply:
For now there is no election reducer or am I missing something? At the moment, the evoting feature still uses the events feature to store the events. As soon as this is no longer the case, we can merge these reducers, I agree :)

Reply:
Oh, I thought we'd moved away from having the events feature store everything about the events. My bad.

Reply:
Not yet, unfortunately. But I will try to look into this as soon as I merged all of the current PRs :)